warehouse/packaging/models.py

```diff
@@ -690,12 +690,31 @@ def urls(self):
         return _urls
 
     def urls_by_verify_status(self, verified: bool):
```

Comment:
Tiny nitpick: perhaps not worth addressing in this PR, but I think `verified: bool` could become a require kwarg to make method calls here slightly comprehensible:

```suggestion
    def urls_by_verify_status(self, *, verified: bool):
```